daxgen/daxgen.py

```diff
@@ -1,9 +1,10 @@
 import networkx as nx
-from Pegasus.DAX3 import ADAG, File, Job, Link
+from Pegasus.DAX3 import ADAG, File, Job, Link, PFN
 
 
-# Set of attributes unique to file nodes. Do not have to be complete,
-_FILE_ATTRS = {'lfn', 'pfns'}
+# Set of mandatory attributes unique to file nodes. Do not have to be
+# exhaustive.
+_FILE_ATTRS = {'lfn'}
```

Comment:
`lfn` stands for _logical file name_ and, similarly, `pfn` stands for _physical file name_. _Pegasus_ documentation uses this shortcuts routinely and I wasn't able to come up with something equally concise but more meaningful.